File: 3st/Hyeonseo/programmer/배열의 평균값.cpp
#include <string>
#include <vector>

using namespace std;

double solution(vector<int> numbers) {
    double answer = 0;
    int length = numbers.size();

    for (int i = 0; i < length; i++) {
        answer = answer + numbers[i];
    }
    
    return answer / length;
}

File: 4st/Jisoo/programmer/배열의 평균값.java
class Solution {
    public double solution(int[] numbers) {
        double answer = 0;
        double sum=0;
       
        for(int i=0;i<numbers.length;i++){
            sum+=numbers[i];
        }
        answer=sum/numbers.length;
        return answer;
    }
}